third_party/utf8_range/utf8_range_neon.inc

```diff
@@ -0,0 +1,208 @@
+#include <arm_neon.h>
+
+/*
+ * Map high nibble of "First Byte" to legal character length minus 1
```

Comment:
Internally when submitting x86 version we changed the comments so that they are more understandable to several people who reviewed the code. I think having a slightly different set of comments in another file might be confusing. I suggest deleting comments here and refer to the sse version of file

It does not mean your comments are bad or anything wrong with them, thank you for doing the work anyway! It's more for the clarity of the future reader when we intend to change anything

Reply:
Indeed. I thought about leaving this file without comment and only reference the sse code.
One catch is the adjustment code is different from sse (neon table lookup is more convenient in this case).
I will delete similar comment and leave only the adjustment related.